packages/langium-cli/package.json

```diff
@@ -34,7 +34,8 @@
     "fs-extra": "^9.1.0",
     "jsonschema": "^1.4.0",
     "langium": "0.1.0",
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
+    "yesno": "^0.3.1"
```

Comment:
NodeJs has the built-in `readline` package. Shall we use that instead of pulling another dependency?

See an example here:
https://github.com/eclipse/openvsx/blob/6674bf4719174b8ba6b6582ef533419caa1d3174/cli/src/util.ts#L158-L182